<launch>
    <!-- Node that performs black line following -->
    <node pkg="your_package_name" type="robust_line_follower.py" name="dynamic_black_line_follower" output="screen">
        <!-- Parameters for dynamic behavior -->
        <param name="forward_speed"    value="0.12" />
        <param name="search_speed"     value="0.3" />
        <param name="crop_fraction"    value="0.5" />
        <param name="max_no_line_time" value="0.5" />
        <param name="kp"               value="0.004" />
        <param name="kd"               value="0.002" />
        <param name="max_ang_vel"      value="0.20" />
        <param name="min_contour_area" value="500" />
        <param name="delta_floor"      value="50" />
    </node>
</launch>
